content/log/index.md

```diff
@@ -0,0 +1,135 @@
+* `service` : name of the service owning the log
+* `source` : \/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/
+* `sourcecategory` : \/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/
+
+### Examples
+#### File log collection configuration
+If you want to gather you nginx logs for instance stored in **/var/log/access.log** and **/var/log/error.log** you would edit your `nginx.yaml` file as follow
+
+{{< highlight yaml >}}
+init_config:
+instances:
+(...)
+#Log section
+logs:
+  - type: file
+    path: /var/log/access.log
+    appname: nginx
+    source: nginx
+    sourcecategory: http_access
+    tags: env:prod 
+
+  - type: file
+    path: /var/log/error.log
+    appname: nginx
+    source: nginx
+    sourcecategory: http_error
+{{< /highlight >}}
+
+#### TCP/UDP Log collection configuration
+
+If your java application doesn't log into a file but forwards its logs on the local 10514 port in TCP edit your `jmx.yaml` file as such in order to listen to this port and forward those logs to your datadog application.:
+
+{{< highlight yaml >}}
+init_config:
+instances:
+(...)
+#Log section
+logs:
+    - type: tcp
+      port: 10514
+      appname: java
+      source: java
+
+{{< /highlight >}}
+
+## Collect log from a custom log file
+
+To send custom log file that would not be part of an existing integration. Rename `custom-logs.yaml.example` in `custom-logs.yaml` and edit the desired parameters as seen in the previous section.
+
+{{< highlight yaml >}}
+logs:
+  - type: file
+    path: /path/to/your/file.log
+    appname: name_of_your_app
+    tags: my_tag_name:my_tag_value
+    source: source_name
+{{< /highlight >}}
+
+## Reserved attributes 
+
+This section is relevant only if you log are JSON formated.
+If so, you need to be aware that some JSON attributes have a specific interaction with your datadog application:
+
+### `date` attribute
+By default your datadog application generates a timestamp for your lag that corresponds to the reception date. This field is used to display your data over the timeline.
+
+But if your log contain a date reserved attribute, then its value is considered as the official log timestamp, date reserved attributes are:
+
+* `timestamp`
+* `date`
+* `_timestamp`
+* `Timestamp`
+* `eventTime`
+* `published_date`
+
+Define any attribute as the official timestamp of your logs with the [log date remapper processor](/log/processing/#log-date-remapper)
+<div class="alert alert-info">
+The recognized date formats are: <a href="https://www.iso.org/iso-8601-date-and-time-format.html">ISO8601</a>, <a href="https://en.wikipedia.org/wiki/Unix_time">UNIX (the milliseconds EPOCH format)</a>  and <a href="https://www.ietf.org/rfc/rfc3164.txt">RFC3164</a>.
+</div>
+
+### `message` attribute
+
+By default, your datadog application considers the `message` attribute as the content to display in priority in your [log list](/log/explore/#log-list).
```

Comment:
Sorry but "your datadog application" sounds weird to me. :)

CHANGE TO

"Datadog" (everywhere if you can)

Reply:
"as the content to display in priority in your [log list](/log/explore/#log-list)."

Change to:

"as the main content of the log. It has then a special display in the Log Explorer and you can run full text search on it."

Reply:
👍